crates/runtime/src/tools/builtin/table_schema.rs

```diff
@@ -44,6 +62,71 @@ impl TableSchemaTool {
             description,
         }
     }
+
+    pub async fn get_schema(
```

Comment:
This refactor allows us to use `TableSchemaTool` in the code, when we only have a `Datafusion` object, and still get the telemetry.